<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_class</class>
 <widget class="QMainWindow" name="view_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>881</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Student ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>First name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Birth date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Social ID</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="classlineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CLASS ID:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
